packages/core/package.json

```diff
@@ -21,9 +21,7 @@
   "copyright.owner": "IBM Corp.",
   "license": "MIT",
   "dependencies": {
-    "@loopback/context": "^4.0.0-alpha.31",
-    "lodash": "^4.17.4",
```

Comment:
- Was lodash supposed to be here? (I can't remember, but I think it was used somewhere at one point)
- Why is it being removed now?

Reply:
Those dependencies weren't being used at all. Not really a part of the task, but I did it anyway